src/xdocs/config.xml

```diff
@@ -379,9 +379,10 @@
 
       <subsection name="Metadata" id="Checker_Metadata">
         <p>
-          Used to keep metadata in the Checkstyle configuration file. This
+          Use to keep metadata in the Checkstyle configuration file. This
```

Comment:
This should be made into a full sentence. Either "This tag is used..." or "Use metadata tags to..." or ....

Reply:
@NitishKumar06 There is a spelling mistake.
> This tag is udse to keep metadata in the Checkstyle configuration file

`udse` => `used`